package.json

```diff
@@ -24,8 +24,8 @@
     "rollup-plugin-json": "^3.1.0",
     "rollup-plugin-node-resolve": "^4.0.0",
     "tape": "^4.9.1",
-    "tv4": "^1.3.0",
     "terser": "^3.11.0",
+    "tv4": "^1.3.0",
```

Comment:
This rearrangement makes sense but also caused a merge conflict. These kinds of orthogonal changes to the main intent of the PR are best included as a separate PR altogether.

Reply:
This might have been yarn. I'll rebase.